### PR TITLE
[EDIT] CSS Background: background-color - Add media to show output

### PR DESCRIPTION
**Pull request by myiphokodo**
### Description

- Added images to show output of each code block
- Provided alt text for images
- Closes issue #3534

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by dakshdeepHERE — APPROVED** [review]
Looks good to me

File: content/css/concepts/background/terms/background-color/background-color.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ h1 {
 }
 ```
 
+![Image showing h1 element with green background over an aquamarine background](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-background-color-example-1.png)
```
Comment (ishg-153) [inline content/css/concepts/background/terms/background-color/background-color.md:43]:
```suggestion
![Shows h1 element with green background over an aquamarine background](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-background-color-example-1.png)
```

File: content/css/concepts/background/terms/background-color/background-color.md [hunk 2]
```diff
@@ -49,3 +51,6 @@ h1 {
   background-color: transparent;
 }
 ```
+
+![Image showing h1 element with transparent background over an aquamarine background](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-background-color-example-2.png)
```
Comment (ishg-153) [inline content/css/concepts/background/terms/background-color/background-color.md:55]:
```suggestion
![Shows h1 element with transparent background over an aquamarine background](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-background-color-example-2.png)
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@myiphokodo I suggested minor changes for the `alt-text`.

Also, remove the text below `h1` that reads "The background is set to aquamarine for contrast." from both the output images.

**Comment by myiphokodo** [comment]
Hello @ishg-153, I've made the suggested edits. Please let me know if there is anything else I should do :)

**Review by ishg-153 — APPROVED** [review]
@myiphokodo I just changed your output images to higher quality ones to expedite the process. This is ready to merge.

**Pull request merged**